src/omero_cli_render.py

```diff
@@ -507,8 +520,11 @@ def test(self, args):
             # img._closeRE()
 
     def test_per_pixel(self, client, pixid, force, thumb):
+        ctx = {'omero.group': '-1'}
         fail = {"omero.pixeldata.fail_if_missing": "true"}
+        fail.update(ctx)
         make = {"omero.pixeldata.fail_if_missing": "false"}
+        make.update(ctx)
```

Comment:
Are these `fail` and `make` contexts actually used somewhere?

Reply:
Yes. Down below what calling `setPixelsId()` under both conditions.

Reply:
Ah of course, sorry, missed that.